src/board_controller/ant_neuro/ant_neuro.cpp

```diff
@@ -297,7 +309,26 @@ int AntNeuroBoard::config_board (std::string config, std::string &response)
             return (int)BrainFlowExitCodes::INVALID_ARGUMENTS_ERROR;
         }
     }
+    else if (config.find (rv_prefix) != std::string::npos)
+    {
+        double new_reference_range = 0.0;
+        std::string value = config.substr (rv_prefix.size ());
+        try
+        {
+            new_reference_range = std::stod (value);
+        }
+        catch (...)
+        {
+            safe_logger (spdlog::level::err, "format is '{}value'", rv_prefix.c_str ());
+            return (int)BrainFlowExitCodes::INVALID_ARGUMENTS_ERROR;
+        }
+
+        // Set the reference_voltage value, assuming you have a class member called reference_voltage
+        reference_range = new_reference_range;
```

Comment:
For sampling rates we check that provided value is correct using GetSamplingRatesAvailable method(check above), is there smth like that for reference voltage? If yes, lets add it here too